released/packages/coq-kruskal-almostfull/coq-kruskal-almostfull.1.0/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+name: "Coq-Kruskal-AlmostFull"
+synopsis: "Base Coq library for manipulating Almost Full relations"
+description: """
+   This library formalizes ground results about Almost Full relations (AF) in Coq 8.14+, up to Dickson's lemma.
+"""  
+maintainer: ["Dominique Larchey-Wendling (https://github.com/DmxLarchey)"] 
+authors: "Dominique Larchey-Wendling (https://github.com/DmxLarchey)"
+license: "MPL-2.0"
+homepage: "https://github.com/DmxLarchey/Kruskal-AlmostFull/"
+bug-reports: "https://github.com/DmxLarchey/Kruskal-AlmostFull/issues"
+dev-repo: "git+https://github.com:DmxLarchey/Kruskal-AlmostFull/"
+
+build: [
+  [ make "-j%{jobs}%" "type"]
+]
+install: [
+  [make "install"]
+]
+
+depends: [
+  "coq" {>= "8.14" & < "8.21~"}
```

Comment:
```suggestion
  "coq" {>= "8.14" & < "8.20~"}
```
Coq 8.20 has not even branched yet, so I don't it makes sense to state compatibility with it.